### PR TITLE
Standardization of tooling 

### PR DESCRIPTION
**Pull request by westeezy**
- Upgrade Flow
- Upgrade `@krakenjs/grumbler-scripts`
- Use GH Actions for CI and Publish
- Use `semantic-release`, `husky`, `commitlint`, for publishing
- Update Repo Badges
- Update Repo License

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2016 PayPal
```
Comment (wsbrunson) [inline LICENSE:178]: do we need to update this?

**Review by wsbrunson — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (westeezy) [reply LICENSE:178]: what year is it?

Comment (westeezy) [reply LICENSE:178]: i synced with checkout-components which i forgot we need to update. one sec we just did this dance recently and i have a better license to use

Comment (wsbrunson) [reply LICENSE:178]: okay cool, i think i did the same thing when i did grabthar

**Pull request merged**